src/vorta/scheduler.py

```diff
@@ -430,16 +430,32 @@ def notify(self, result):
         profile_id = result['params']['profile'].id
 
         if result['returncode'] in [0, 1]:
-            notifier.deliver(
-                self.tr('Vorta Backup'),
-                self.tr('Backup successful for %s.') % profile_name,
-                level='info',
-            )
-            logger.info('Backup creation successful.')
-            # unpause scheduler
-            self.unpause(result['params']['profile_id'])
+            try:
+                notifier.deliver(
+                    self.tr('Vorta Backup'),
+                    self.tr('Backup successful for %s.') % profile_name,
+                    level='info',
+                )
+                logger.info('Backup creation successful.')
+                # unpause scheduler
+                self.unpause(result['params']['profile_id'])
 
-            self.post_backup_tasks(profile_id)
+                self.post_backup_tasks(profile_id)
+                # Notify after successful post_backup_tasks
+                notifier.deliver(
+                    self.tr('Vorta Backup'),
+                    self.tr('Post Backup Tasks successful for %s' % profile_id),
+                    level='info',
+                )
+                logger.info('Post Backup Tasks successful for %s' % profile_id)
```

Comment:
This is log line is already produced in `post_backup_tasks`. Maybe all the code you added would better fit there.

Reply:
@real-yfprojects I have added the line under post_backup_tasks